tests/serialization/test_object_template.py

```diff
@@ -20,20 +20,20 @@ def test_object_template_serde():
     color_template = PropertyTemplate("Color", bounds=CategoricalBounds(["red", "green", "blue"]))
     # Properties are a mixture of property templates and [template, bounds], pairs
     block_template = MaterialTemplate("Block", properties=[[length_template, sub_bounds],
-                                                           color_template])
+                                                           [color_template]])
```

Comment:
I don't think this is a desirable change--the user should be able to specify the template on its own, without wrapping it in brackets